components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
```

Comment:
This should be using "@hods/sass-base" rather than "@not-govuk/sass-base".

Reply:
Updated in latest commit